src/domain/interface.ts

```diff
@@ -262,6 +263,7 @@ export interface Application {
   sections: Sections;
   updates: ApplicationUpdate[] | UserViewApplicationUpdate[];
   approvedAppDocs: ApprovedAppDocument[];
+  attestationByUtc?: Date; // calculated from approvedAtUtc
```

Comment:
I think it's better to have it as `attestationByUtc: Date  | undefined`  ie always return it in the response

Reply:
same for application summary?